scripts/client.js

```diff
@@ -1,47 +1,72 @@
+/* @flow */
+
 import KintoClient from "kinto-client";
 import endpoint from "kinto-client/lib/endpoint";
 import { isHTTPok } from "./utils";
 
 
-let client;
+let client: ?KintoClient;
 
-function getAuthHeader(session) {
-  const {authType, credentials} = session;
+function getAuthHeader(session: Object): ?string {
+  const {authType, credentials}: {
+    authType: string,
+    credentials: Object
+  } = session;
   switch(authType) {
     case "fxa": {
-      const {token} = credentials;
+      const {token}: {token: string} = credentials;
       return "Bearer " + token;
     }
     case "basicauth": {
-      const {username, password} = credentials;
+      const {username, password}: {
+        username: string,
+        password: string,
+      } = credentials;
       return "Basic " + btoa([username, password].join(":"));
     }
   }
 }
 
-export function setupClient(session) {
-  const {server} = session;
+export function setupClient(session: Object): KintoClient {
+  const {server}: {server: string} = session;
```

Comment:
For some of these duplicated types, it seems like it would be better to just move the type into the function declaration, i.e.

```javascript
export function setupClient(session: {server: string}): KintoClient {
  const {server} = session;
```